### DISCUSSION TITLE
[Question] Does ORY Hydra v0.10.8 support MySQL 8

### DISCUSSION THREAD

**Discussion by cuong-linkbal** [ory/.github#123] [Q&A] [answered]
Hi, I'm having a very old system use ORY Hydra v0.10.8 and MySQL 5.7 (and Go lang 1.9).  
I want to upgrade MySQL to 8.0. 
But looks like Hydra v0.10.8 does not work well with MySQL 8.  
I can not run `hydra clients create` command.  
So Does ORY Hydra v0.10.8 support MySQL 8 ?  
if not , which versions support ?

Thank you

**Comment by vinckr** [comment, marked as answer]
Hello @cuong-linkbal 

Ory Hydra v0.10.8 is a very old version. The current version of Ory Hydra is much more recent, with the latest stable release being v2.1.2 as of the time of writing this response.
According to the [Ory Hydra documentation on database setup](https://www.ory.sh/docs/hydra/self-hosted/dependencies-environment), Ory Hydra supports MySQL 5.7+ for more recent versions. However, this information is not specific to the v0.10.8 version you are using.
Given that you're using such an old version of Ory Hydra, it's highly likely that it does not support MySQL 8.0. 

I would strongly recommend upgrading Ory Hydra to a more recent version. The [upgrade guide](https://www.ory.sh/docs/hydra/self-hosted/upgrade) provides detailed instructions on how to perform upgrades.

**Reply by cuong-linkbal** [reply]
@vinckr   
Thank you very much for your response.
I understand completely.
